app/javascript/components/editor-config/placeholder.js

```diff
@@ -0,0 +1,26 @@
+import { Plugin } from 'prosemirror-state'
+import { Decoration, DecorationSet } from 'prosemirror-view'
+
+export default () => {
```

Comment:
we should consider avoiding using default and unnamed exports (see: 
https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/)